sphinx/writers/latex.py

```diff
@@ -863,26 +867,31 @@ def visit_desc_parameterlist(self, node: Element) -> None:
         if self.has_tp_list:
             if self.orphan_tp_list:
                 # close type parameters list (#2)
-                self.body.append('}{')
+                self.body.append('}' + CR + '{')
                 # empty parameters list argument (#3)
                 return
         else:
             # close name argument (#1), open parameters list argument (#2)
-            self.body.append('}{')
+            self.body.append('}' + CR + '{')
         self._visit_sig_parameter_list(node, addnodes.desc_parameter)
 
     def depart_desc_parameterlist(self, node: Element) -> None:
         # close parameterlist, open return annotation
-        self.body.append('}{')
+        self.body.append('}' + CR + '{')
 
     def visit_desc_type_parameter_list(self, node: Element) -> None:
         # close name argument (#1), open type parameters list argument (#2)
-        self.body.append('}{')
+        self.body.append('}' + CR + '{')
         self._visit_sig_parameter_list(node, addnodes.desc_type_parameter)
 
     def depart_desc_type_parameter_list(self, node: Element) -> None:
-        # close type parameters list, open parameters list argument (#3)
-        self.body.append('}{')
+        if self.orphan_tp_list:
+            # this node next sibling isn't a desc_parameterlist, there are no parameters:
+            # close the type list, output an empty parameter list, and open return annotation.
```

Comment:
```suggestion
            # close the type list, output an empty parameter list, open return annotation.
```

(shorten the comment line)